crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -66,6 +67,50 @@ fn should_farm_during_initial_plotting() -> bool {
     total_cpu_cores > 8
 }
 
+/// Plotting thread priority
+#[derive(Debug, Parser, Copy, Clone)]
+enum PlottingThreadPriority {
+    /// Minimum priority
+    Min,
+    /// Default priority
+    Default,
+    /// Max priority (not recommended)
+    Max,
```

Comment:
any specific reason why reference implementation wants to use this plotting when its clearly not recommended? Maybe this should not be an option at all ?

Reply:
There is actually a much more fine-grained control available over priority. But for now I decided to give extreme option for users to try and experiment with. It will likely evolve over time, we may end up removing it if not helpful. This might be desirable to accelerate plotting at all costs, but it will likely make farming while plotting not possible at all (though not guaranteed).